sql_export/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import test_sql_query
```

Comment:
in v8, tests are now automatically launched  if they are written under the module 'tests' in a submodule with a the name starting with test_
But before V8, we need to add the mentioned lines to say to Odoo which tests must be launched